.github/workflows/build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Setup Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 2.7.5
+          ruby-version: 2.7.7
```

Comment:
I believe this defaults to using `.ruby-version` so we could leave it out and avoid duplication.

Reply:
yes, that's correct! just reread their docs and says `# Not needed with a .ruby-version file`